packages/docs/blog/2026-02-02-release-26-2-0.md

```
@@ -0,0 +1,158 @@
+authors: matt-fidd
+---
+
+This release contains the following notable improvements, along with numerous fixes:
+
+- Allow multiple dashboard pages
+- Experimental feature: Custom themes [docs](https://actualbudget.org/docs/experimental/custom-themes)
+- Experimental feature: Budget analysis report [docs](https://actualbudget.org/docs/experimental/budget-analysis-report)
+
+<!--truncate-->
+
+**Docker Tag: v26.2.0**
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)